package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@equinor/amplify-components",
-  "version": "3.9.5",
+  "version": "3.9.6",
```

Comment:
Version number shouldn't be bumped when we just change the config files. Because of the github action we have that compares the config files that are in amplify-components and the local ones for that given project, we don't upgrade the amplify-components package but just run the "download config" script (`wget .... | bash`)

Reply:
Makes sense 👀